evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -823,3 +823,14 @@ def test_explain_predictions_stacked_ensemble(problem_type, dummy_stacked_ensemb
 
     with pytest.raises(ValueError, match="Cannot explain predictions for a stacked ensemble pipeline"):
         explain_predictions_best_worst(pipeline, X, y)
+
+
+@pytest.mark.parametrize("estimator", ["Logistic Regression Classifier", "Decision Tree Classifier", "Extra Trees Classifier"])
+def test_explain_predictions_oversampler(estimator, fraud_100):
+    pytest.importorskip('imblearn.over_sampling', reason='Skipping test because imbalanced-learn not installed')
+    X, y = fraud_100
+    pipeline = BinaryClassificationPipeline(component_graph=["Imputer", "One Hot Encoder", "DateTime Featurization Component", estimator])
```

Comment:
@bchen1116 there's no oversampler in this pipeline! I'm guessing you forgot to add it? The other test looks 💯 🔥  haha

Reply:
Wow that's terrible. I'll fix!

Reply:
Lol its ok! This is why I love code review 😁